hibernate-core/src/main/java/org/hibernate/jpa/boot/spi/ProviderChecker.java

```diff
@@ -66,7 +66,7 @@ public static boolean hibernateProviderNamesContain(String requestedProviderName
 
 	/**
 	 * Extract the requested persistence provider name using the algorithm Hibernate uses.  Namely, a provider named
-	 * in the 'integration' map (under the key '{@value AvailableSettings#JPA_PERSISTENCE_PROVIDER}') is preferred, as per-spec, over
+	 * in the 'integration' map (under the key '<b>javax.persistence.provider</b>') is preferred, as per-spec, over
```

Comment:
No possibility to refer to interface string static field. Changed it to explicit string. Guess the key should be stable, though not perfect.

Reply:
`@value` should have worked. What do you use for javadoc generation?

Reply:
```
./gradlew :hibernate-core:javadoc
```